spi/w5500_spi/README.adoc

```diff
@@ -0,0 +1,59 @@
+= Attaching a W5500 via SPI
+
+This example code shows how to interface the Raspberry Pi Pico to the WIZ850io(W5500) board. The particular device used can be interfaced via SPI, we are using SPI, and interfacing at 3.3v.
+
+[NOTE]
+======
+This is a very basic example, and it send and receive data from the WIZ850io(W5500).After you get off the communication command(S"n"_CR) on the WIZ850io, make sure the communicationcommand is clear."n" is socket number.
```

Comment:
Missing space in "communicationcommand"

Reply:
Sorry, I fixed this line

Reply:
See Alasdair's comment above: "This will probably get allocated to an intern over the summer, as we're going to be adding a lot of example code for various breakouts."